UNRELEASED.md

```diff
@@ -13,6 +13,7 @@
 
 - Added an outline on `Banner` for Windows high contrast mode ([#2878](https://github.com/Shopify/polaris-react/pull/2878))
 - Fixed Autocomplete / ComboBox focus ([#1089](https://github.com/Shopify/polaris-react/issues/1089))
+- Fixed coloring issue for button icons (#2958)
```

Comment:
Can this be a proper link please? Llike the line above

Reply:
👍